software/hook0.yml

```
@@ -0,0 +1,11 @@
+name: "Hook0"
+website_url: "https://www.hook0.com"
+source_code_url: "https://gitlab.com/hook0/hook0"
+description: "A Webhooks-as-a-service (WaaS) that makes it easy for online products to provide webhooks. Dispatch up to 3,000 events/month with 7 days of history retention for free."
+licenses:
+  - SSPL-1.0
+platforms:
+  - Rust
+  - Docker
```

```suggestion
platforms:
  - Rust
  - Nodejs
  - Docker
```
As far as I understand from the Documentation, when hosting it without Docker you will need to use Nodejs for the Hook0 UI, correct?

---

Indeed!